airflow/contrib/operators/dataproc_operator.py

```diff
@@ -144,7 +171,6 @@ class DataprocClusterCreateOperator(BaseOperator):
     @apply_defaults
     def __init__(self,
                  cluster_name,
-                 project_id,
```

Comment:
Are you sure this will work for automated documentation generation? I think the doc building with autoapi is failing now because those explicit parameters were removed. They should remain here IMHO and should be used directly in super() method.

Reply:
Done